tests/unit/api/PacbioRun.spec.js

```diff
@@ -121,80 +123,81 @@ describe('Run', () => {
       run.plate.wells[0] = { position: 'A1', libraries: [{ id: 1 }] }
       run.plate.wells[1] = { position: 'A2', libraries: [{ id: 2 }] }
 
-      api = build(Api.Config, process.env)
-      api.traction.pacbio.runs.create = jest.fn()
-      api.traction.pacbio.plates.create = jest.fn()
-      api.traction.pacbio.wells.create = jest.fn()
-      api.traction.pacbio.runs.destroy = jest.fn()
-      api.traction.pacbio.plates.destroy = jest.fn()
-      api.traction.pacbio.wells.destroy = jest.fn()
+      api = build({ config: Api.Config, environment: process.env })
+      pacbioRequest = api.traction.pacbio
```

Comment:
Aww, thought you could dry this out further to just `api.traction.pacbio.runs`, but nope, `api.traction.pacbio` sneaks in at the bottom on its own.

Reply:
Exactly. I will come back to this later anyway. I have seen some great advice recently on testability which would mean you can create a wrapper around the original api with a proxy so you don't have to mock out the individual calls but that is something for another day.